docs/pages/admin-guides/management/guides/scim/scim.mdx

```diff
@@ -0,0 +1,109 @@
+Access List.
+- Access Lists of SCIM type must be created in advance in Teleport. In this guide we're creating them using Terraform.
+- Only Access Lists of type `scim` can be managed by SCIM providers.
+- Role assignments are handled in Teleport, while group membership is delegated to External Identity Management System (like SailPoint).
+
+## Prerequisites
+- Teleport Enterprise v(=teleport.version=)
+- Teleport Terraform provider v(=teleport.version=)
+- A running Teleport cluster with SSO enabled (e.g. Okta SAML connector)
+- Identity Management System (like SailPoint) with SCIM support
+- SCIM Provider with OAuth 2.0 Client Credentials grant type support
+
+## Step 1/3: Create an SCIM-Managed Access List in Teleport
+
+Create a new Access List in Teleport using Terraform. Be sure to set `type = "scim"`
+and match `spec.title` to the name of `displayName` of  SCIM group that we be provided to Teleport:
+
+
+```hcl
+resource "teleport_access_list" "acl-group-requester" {
+  header = {
+    version = "v1"
+    name = "scim-group-requester"
+  }
+  spec = {
+    title = "GroupRequester"
+    type = "scim"
+    grants = {
+      roles  = ["requester"]
+      traits = []
+    }
+    owners = [
+      {
+        name = "alice"
+      }
+    ]
+    membership_requires = {
+      roles = []
+    }
+    ownership_requires = {
+      roles = []
+    }
+    audit = {
+      recurrence = {
+        frequency    = 3
+        day_of_month = 15
+      }
+    }
+  }
+}
+```
+
+<Admonition type="info">
+    The SCIM group name (displayName) in SCIM Provider must exactly match
+    `spec.title` in the Teleport Access List.
+</Admonition>
+
+
+## Step 2/3. Configure SCIM Integration
+Teleport provides a guided web UI based configuration flow for SCIM integration.
+
+In the Teleport Web UI, go to **"Add new integration"** and select **SCIM**.
+
+![Plugin Install](../../../../../img/sailpoint/plugin-install.png)
+
+
+Select the **SAML connector** to associate SCIM-provisioned users with SSO logins.
+
+- By default, SSO users in Teleport are ephemeral.
+- SCIM provisioning ensures users are persistently created and managed by External Identity management system via SCIM protocol.
+
+![Connector Selection](../../../../../img/sailpoint/plugin-connector.png)
+
+
+Click **Continue** to proceed to the **SCIM Credentials** screen.
+
+- Teleport uses OAuth 2.0 Bearer Tokens for SCIM authentication.
+- Copy the **Client ID**, **Client Secret**, and **Base URL** — you'll use them
+  when configuring your Identity Provider in the next step.
+
+### Step 3/3. Configure SCIM integration with your Identity Management SCIM provider
+
+SCIM configuration may differ depending on your IdP. The integration has been officially tested with the following providers:
+
+- [SailPoint Teleport SCIM connector setup](./sailpoint.mdx)
+- In case of Other SCIM providers, please refer to their documentation for SCIM integration.
```

Comment:
```suggestion
SCIM configuration may differ depending on your IdP. The integration has been officially tested with the following providers:

- [SailPoint Teleport SCIM connector setup](./sailpoint.mdx)

In case of other SCIM providers, please refer to their documentation for setting up a SCIM integration.
```
Grammar/style fix